### DISCUSSION TITLE
Where is POETRY_NO_INTERACTION documented?

### DISCUSSION THREAD

**Discussion by piotrekkr** [python-poetry/poetry#8977] [General] [answered]
Hello.

I'm checking how to use poetry with docker and I noticed everyone is using `POETRY_NO_INTERACTION=1` env variable with docker (like in here https://github.com/orgs/python-poetry/discussions/1879). In the same Dockerfile examples I can see somehing like `RUN poetry install --only main --no-interaction --no-ansi` etc. Seems like `--no-interaction` is redundant. But on the other hand I could not find this environment variable defined or documented anywhere in this project.

Is this everyone copy pasting someone else and this env var does not do anything or I just missed it? 
Thanks.

**Comment by dimbleby** [comment, marked as answer]
it is not a thing, this is cargo-culting
